lib/mongoid_session_store/store.rb

```diff
@@ -9,7 +9,7 @@ class Store < ActionDispatch::Session::AbstractStore
   private
 
     def generate_sid
-      BSON::ObjectId.new.to_s
+      SecureRandom.hex(32)
```

Comment:
The math behind this is pretty interesting: https://en.wikipedia.org/wiki/Birthday_attack 

If we were to use a 64 bit hex instead it would be extremely improbable to get a collision, but for 32 bit it might happen after just a few thousand stored sessions (Not sure how many active sessions we store simultaneously, but I guess it's far less?)